code/src/styles/AddTodoForm.css

```diff
@@ -0,0 +1,16 @@
+
+/* I didn't know how to style the datepicker 
+with styled components (since it is an automatically created class), 
+so I put it in a css-file instead.
+Haven't figured out how to style the 
+input/placeholder so that 'Due Date' is displayed
+on the datepicker (just as in the select dropdown)*/
```

Comment:
maybe this article could help here: 
https://www.hongkiat.com/blog/jquery-ui-datepicker/

Reply:
Oh! Very nice of you to look this up and provide a link! 🥇 Will check it